test/benchmark/benchmark_helper.rb

```diff
@@ -0,0 +1,29 @@
+require 'benchmark/ips'
+
+require File.expand_path('../../../config/environment', __FILE__)
+require 'factory_girl_rails'
+
+ unless Rails.env.production? and ActiveRecord::Base.connection.class == ActiveRecord::ConnectionAdapters::SQLite3Adapter and Rails.configuration.database_configuration[Rails.env]['database'] == ":memory:"
+  require 'pry'; binding.pry
+  puts "Not properly configured for benchmark, please add this to your config/database.yaml:"
+  puts <<EOS
+production:
+  adapter: sqlite3
+  database: ":memory:"
+  migrate: false
+  pool: 1
+  timeout: 1000
+EOS
+  exit 1
+end
```

Comment:
`FOREMAN_NO_DEPRECATIONS` does what exactly? I don't see any instance of it by grepping the dir.